constant.py

```diff
@@ -10,6 +10,11 @@
 JW_PLAYER="JWPLAYER"
 ADPOD="ADPOD"
 
+LI_SPONSORSHIP = "SPONSORSHIP"
```

Comment:
We can rename `LI_SPONSORSHIP` to `LI_TYPE`. It would be readable.

Reply:
Will rename it to LI_TYPE_SPONSORSHIP